tests/Console/ModelsCommand/GeneratePhpdocWithFqn/Models/Post.php

```diff
@@ -6,20 +6,30 @@
 
 use Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\GeneratePhpdocWithFqn\Casts\CastType;
 use Eloquent;
-use Illuminate\Database\Eloquent\{
-    Builder as EloquentBuilder,
-    Collection,
-    SoftDeletes
-};
+use Illuminate\Database\Eloquent\Builder as EloquentBuilder;
+use Illuminate\Database\Eloquent\Collection;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\HasMany;
+use Illuminate\Database\Eloquent\SoftDeletes;
 use Illuminate\Database\Query\Builder as QueryBuilder;
 use Illuminate\Support\Carbon;
 
 class Post extends Model
 {
     use SoftDeletes;
 
+    /**
+     * Special hack to avoid code style fixer removing unused imports
+     * which play a role when generating the snapshot
+     */
```

Comment:
Not the proudest thing to to but IMHO acceptable for a single exception here, as the imported (and not-yet-used use statements) are (only) relevant in this test